### PR TITLE
nodejs-ca: Add download file - index.md

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR is related to [#1197](https://github.com/nodejs/nodejs.org/issues/1197)

I have created the '**Download**' index.md translation to catalan, according Node.js Website translation Policy indications.

nodejs-ca: Add download file - index.md

### CODE REVIEW

File: locale/ca/download/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: download.hbs
+title: Descàrrega
+download: Descàrrega
+downloads:
+    headline: Descàrregues
+    lts: LTS
+    current: Actual
+    tagline-current: Últimes característiques
+    tagline-lts: Recomennat per a la majoria
```
Comment (jose-lara) [inline locale/ca/download/index.md]: "Recomanat" instead of "Recomennat"

File: locale/ca/download/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    display-hint: Mostrar descàrregues per a
+    intro: >
+        Descarregui el codi font de Node.js o un instal·lador pre-compilat per a la seva plataforma, i comenci a desenvolupar avui.
+    currentVersion: Versió actual
+    buildInstructions: Building Node.js from source on supported platforms
+additional:
+    headline: Plataformes addicionals
+    intro: >
+        Membres de la comunitat de Node.js proveén paquets pre-compilats de forma no oficial per a plataformes addicionals no suportades per l'equip central de Node.js que poden no estar al mateix nivell de les versions actuals oficials de Node.js.
```
Comment (jose-lara) [inline locale/ca/download/index.md]: "proveeixen" instead of "proveén"

**Review by jose-lara — COMMENTED** [review]
I added two modifications for that

**Comment by fhemberger** [comment]
@Tsuna-mi I'd love to get this merged. Can you please look at @jose-lara's feedback?

**Comment by Tsuna-mi** [comment]
Changes applied, also ready to merge.
Thank you so much @jose-lara & @fhemberger!

**Comment by fhemberger** [comment]
Wooop! Awesome!

**Pull request merged**